Detox.js

```diff
@@ -84,6 +84,7 @@ let waitFor;
  * * `reloadReactNative` - should be enabled for React Native applications.
  * * `reuse` - reuse application for tests. By default, Detox reinstalls and relaunches app.
  * * `registerGlobals` - (default: true) Register Detox helper functions `by`, `element`, `expect`, `waitFor` globally.
+ * * `url` - a URL to open on whenever the app is launched (android) or immediately afterwards (iOS). Useful for opening a bundle URL at the beginning of tests when working with Expo.
```

Comment:
Which were you thinking:
- `opening a deep-link, or a bundle URL when working with Expo, at the beginning of tests.`, or
- `a URL to open via deep-link whenever the app is launched`

Reply:
@kobenguyent Running that produced a very big diff for me in the README, is that normal?

Reply:
I think that might make sense for now 👍 

Not sure if it has to do with line endings (strange it wouldn't affect every line if that were the case), but this is the diff in case you were wondering: [diff.patch](https://github.com/user-attachments/files/16603313/diff.patch)